content/md/first-steps/manage-your-attributes.md

```diff
@@ -9,73 +9,29 @@ related: what-is-an-attribute
 
 # Overview
 
-To fit the PIM application to your needs, Akeneo enables you to define attributes. For example, if you need to capture data about the weight of your products, you can add an attribute type as ‘metric’.
+To fit the PIM application to your needs, Akeneo enables you to define attributes. Akeneo offers you 14 types of attributes on the Community Edition version, and 15 types on the Enterprise Edition version. The additional one is the Asset type, which is part of our [Product Asset Manager feature] (/articles/work-with-assets.html).
 
-You will be able to select the most appropriate attribute type for your needs (see [Create an attribute](#create-an-attribute)). For example, you can choose to create a field as a text, in which users can write what they want, or as a drop down list (simple or multi select) with a selection of values, forcing users to select from a list of predefined options.
+For example, if you need to capture data about the weight of your products, you can add an attribute type as ‘metric’.
```

Comment:
Weird to have an example since you have another one just below.
I will move this sentence after the next one and begin it with: "Another example would be..."

Reply:
I have removed the example above.